tool/src/test/resources/org/datacommons/tool/genmcf/resolution/output/report.json

```diff
@@ -125,50 +124,28 @@
     "measurementMethod": "",
     "observationPeriod": "",
     "scalingFactor": "",
-    "unit": "",
-    "seriesLargestPercentDiff": {
-      "percentDifference": 0.0
-    }
+    "unit": ""
   }, {
-    "placeDcid": "geoId/0649670",
-    "statVarDcid": "l:SVId",
+    "placeDcid": "wikidataId/Q1353",
+    "statVarDcid": "CumulativeCount_MedicalTest_ConditionCOVID_19_Positive",
     "measurementMethod": "",
     "observationPeriod": "",
     "scalingFactor": "",
-    "unit": "",
-    "seriesLargestPercentDiff": {
-      "percentDifference": 0.0
-    }
+    "unit": ""
   }, {
```

Comment:
Can we drop the entry if there are no errors?

Reply:
done